test/k8sT/manifests/hubble-cli.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: hubble-cli
+spec:
+  selector:
+    matchLabels:
+      k8s-app: hubble-cli
+  template:
+    metadata:
+      labels:
+        k8s-app: hubble-cli
+    spec:
+      containers:
+        - name: hubble-cli
+          image: "quay.io/cilium/hubble:v0.5.0"
```

Comment:
are w going to also test hubble cli in e2e tests? if yes we might want to create a chart with image version parametrized. No need to do it as part of this PR though.

Reply:
Very good point actually - we are likely going to invoke the Hubble CLI from other e2e tests as well in the future. So it absolutely makes sense to have the `hubble-cli` containers for the Hubble client available in the Cilium helm charts.